src/index.html

```diff
@@ -1,5 +1,8 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
```

Comment:
The HTML tag should not have attributes split across multiple lines unless there are more than two attributes or the attributes are long. Consider placing them on a single line.